crates/clarirs_py/Cargo.toml

```diff
@@ -16,6 +16,7 @@ pyo3 = { version = "0.23.4", features = [
     "num-bigint",
 ] }
 clarirs_core = { path = "../clarirs_core", features = ["runtime-checks"] }
+clarirs_num = { path = "../clarirs_num" }
```

Comment:
I assume this was added to convert directly between BitVecError and PyBitVecError? I think for consistency, BitVecErrors should probably be converted first to ClarirsErrors in clarirs_core, and then converted again to ClaripyErrors in clarirs_py. This way there would be no direct depenedency between clarirs_py and clarirs_num